libfwupdplugin/fu-firmware.c

```diff
@@ -676,7 +676,7 @@ fu_firmware_get_bytes_with_patches(FuFirmware *self, GError **error)
  * Since: 1.6.0
```

Comment:
Update to 2.0.0 with new return value

Reply:
I'm not sure; whist true from a strict ABI point of view, I think the libfwupdplugin Since tags are more useful for "can I backport this to branch 1_X_Y" -- and in this case the method accepts the same set of parameters, just as an enum rather than a define.